input/templates/my-account-change-password.hbs

```diff
@@ -18,42 +18,52 @@
           <div class="my-account-content">
             <div class="row">
               <div class="col-sm-3">
-                {{> myaccount/my-account-sidebar positionSeven=true}}
+                {{> myaccount/my-account-sidebar changeMyPasswordTab=true}}
               </div>
               <div id="my-account-desktop-content" class="col-sm-9">
-                <form id="form-change-password" action="{{@root.meta._links.changePassword.href}}" method="POST">
-                  {{> form/global-messages messages=content.changePassword.feedbackMessage}}
-                  <input type="hidden" name="csrfToken" value="{{@root.meta.csrfToken}}"/>
-                  <input type="hidden" name="id" value="{{content.address.id}}"/>
-                  <div class="change-password">
-                    <div class="change-password-wrapper">
-                      <div class="personal-details-title">
-                        <span>{{i18n "my-account:changePassword.changePasswordTitle"}}</span>
-                        <span class="pull-right required-text">{{i18n "main:form.required"}}*</span>
+                <div class="change-password">
+                  <div class="change-password-wrapper">
+                    <div class="personal-details-title">
+                      <span>{{i18n "my-account:changePassword.changePasswordTitle"}}</span>
+                      <span class="pull-right required-text">{{i18n "main:form.required"}}*</span>
+                    </div>
+                    <form id="form-change-password" action="{{@root.meta._links.changePassword.href}}" method="POST">
+                      <input type="hidden" name="csrfToken" value="{{@root.meta.csrfToken}}"/>
+                      {{> form/global-messages messages=content.changePasswordForm.messages}}
+                      {{> form/global-errors errors=content.changePasswordForm.errors}}
+                      <div class="row">
+                        <div class="col-sm-6">
+                          <div class="form-sections">
+                            <span class="form-labels">{{i18n "my-account:changePassword.oldPasswordLabelTxt"}}*</span><br>
+                            <input name="oldPassword" class="form-inputs" type="password">
```

Comment:
I have added the missing "old password" field, otherwise without it anybody with a logged in account can change the password and that becomes a great security thread.